apitest/src/main/java/bisq/apitest/linux/BitcoinDaemon.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.apitest.linux;
+
+import java.io.IOException;
+
+import lombok.extern.slf4j.Slf4j;
+
+import static bisq.apitest.linux.BashCommand.isAlive;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
+
+
+import bisq.apitest.config.ApiTestConfig;
+
+// Some cmds:
+// bitcoin-cli -regtest generatetoaddress 1 "2MyBq4jbtDF6CfKNrrQdp7qkRc8mKuCpKno"
+// bitcoin-cli -regtest getbalance
+// note:  getbalance does not include immature coins (<100 blks deep)
+// bitcoin-cli -regtest getbalances
+// bitcoin-cli -regtest getrpcinfo
+
+@Slf4j
+public class BitcoinDaemon extends AbstractLinuxProcess implements LinuxProcess {
+
+    public BitcoinDaemon(ApiTestConfig config) {
+        super("bitcoind", config);
+    }
+
+    @Override
+    public void start() throws InterruptedException, IOException {
+        String bitcoindCmd = "export LD_LIBRARY_PATH=" + config.berkeleyDbLibPath + ";"
+                + " " + config.bitcoinPath + "/bitcoind"
+                + " -datadir=" + config.bitcoinDatadir
+                + " -daemon";
```

Comment:
`export LD_LIBRARY_PATH=` doesn't seem to be necessary with my distribution of `bitcoind`. Also, `config.bitcoinPath` seems to expect to be pointed to the `bin` sub-directory of bitcoin-core's home (the one that contains the `bitcoind` executable). That's a bit unexpected. I think it's more intuitive for it to point to the executable directly, or alternatively to bitcoin-core's home directory.

Reply:
`export LD_LIBRARY_PATH` is necessary for my bitcoin-core installation.  This issue is something we'll need to sort out for an unknown variety of setups.   

I think `config.bitcoinPath` should point to the directory containing bitcoin-core binaries, not the `bitcoind` and `bitcoin-cli` binaries themselves -- requiring two `apitest` config options instead of one.   On the other hand, my `bitcoin-core` binaries are isolated in my `bitcoin-core` src directory (I never ran `make install`), not a default system `bin` directory where many unrelated executables are installed.  Let's keep discussing...

Reply:
You no longer need to configure a `BerkeleyDb` lib path if your bitcoin-core binary is statically linked to `BerkeleyDb`.  The `berkeleyDbLibPath` option now defaults to an empty string.  If not set, no `export LD_LIBRARY_PATH` will not be pre-pended to the `bitcoind` command.